jumpscale/packages/admin/actors/packages.py

```diff
@@ -1,5 +1,5 @@
 from jumpscale.servers.gedis.baseactor import BaseActor, actor_method
-from jumpscale.loader import j
+from jumpscale.god import j
```

Comment:
why  switching the loader?

Reply:
Didn't change that don't know what happened by merge or something. anyway reverted